tests/ClientTest.php

```diff
@@ -210,6 +207,24 @@ public function join($timeout = null)
     }
 }
 
+class Dummy_Serializer implements \Raven\SerializerInterface
```

Comment:
> we should check if the serializer is correctly set

Yes. This is exactly what I\'m testing. Look at this https://github.com/getsentry/sentry-php/pull/454/files#diff-668268a1040801a44451100bf9062b17R2510
I set some serializer and then I check if it has been set